src/routes/search.ts

```diff
@@ -1,10 +1,9 @@
 import { Router } from 'express';
 import * as searchController from '../controllers/searchController';
-import * as stallController from '../controllers/stallController';
 
 const router = Router();
 
-router.get('/', stallController.indexStallFuncs);
+router.get('/', searchController.emptySearchFuncs);
```

Comment:
Will it work if I do 
```suggestion
router.get('/', searchController.searchFuncs);
```
since the search still works on empty input.